controllers/configmap_controller.go

```diff
@@ -219,7 +219,7 @@ func (r *ConfigMapReconciler) ensureInstancesAreUpToDate(instances []*instances.
 func (r *ConfigMapReconciler) deconfigureInstances(instances []*instances.InstanceInfo, nodes *core.NodeList) error {
 	for _, node := range nodes.Items {
 		// Only looking at BYOH nodes
-		if _, present := node.Annotations[BYOHAnnotation]; !present {
+		if _, present := node.GetLabels()[BYOHLabel]; !present {
```

Comment:
I suggest updating the filter when you are getting the list of nodes.

Reply:
Done.